identity.go

```diff
@@ -37,7 +37,7 @@ func newIdentity(config config, userIDString string) (identity, error) {
 	generatedAppID := newAppId(config.AppSecret)
 
 	if !bytes.Equal(generatedAppID, config.AppID) {
-		return identity{}, errors.New("App secret and app ID mismatch")
+		return identity{}, errors.New("app secret and app ID mismatch")
```

Comment:
Maybe a rule to specify in the Go code manifesto?

Reply:
Might be indeed, don't really know.
This is mostly for consistency with other libs and projects, and for readability when the error is wrapped in other outputs.
I'll add that so it's up for discussion